integration-cli/docker_api_containers_test.go

```diff
@@ -1403,18 +1403,6 @@ func (s *DockerSuite) TestPostContainersCreateShmSizeNegative(c *check.C) {
 	c.Assert(string(body), checker.Contains, "SHM size must be greater then 0")
 }
 
-func (s *DockerSuite) TestPostContainersCreateShmSizeZero(c *check.C) {
```

Comment:
No. ShmSize = 0 was an invalid value earlier. Now we interpret it as default 64M.